test/dbservice/groups_test.exs

```diff
@@ -8,11 +8,28 @@ defmodule Dbservice.GroupsTest do
 
     import Dbservice.GroupsFixtures
 
-    @invalid_attrs %{input_schema: nil, locale: nil, locale_data: nil}
+    @invalid_attrs %{
+      name: nil,
+      type: nil,
+      program_type: nil,
+      program_sub_type: nil,
+      program_mode: nil,
+      program_start_date: nil,
+      program_target_outreach: nil,
+      program_donor: nil,
+      program_state: nil,
+      batch_contact_hours_per_week: nil,
+      group_input_schema: nil,
+      group_locale: nil,
+      group_locale_data: nil
+    }
 
     test "list_group/0 returns all group" do
       group = group_fixture()
-      assert Groups.list_group() == [group]
+      [head | _tail] = Groups.list_group()
+      assert is_list(Groups.list_group()) == is_list([group])
+      assert Map.has_key?(head, :program_type) == Map.has_key?(group, :program_type)
```

Comment:
this looks like we're comparing two boolean values. I don't think it's a valid check.

As I understand from docs, `Map.has_key` returns a boolean value
https://hexdocs.pm/elixir/1.12/Map.html#has_key?/2

So in a way, it looks like this check is `boolean == boolean`, which doesn't seem right.

What was the purpose of this check?